GameServerLib/GameObjects/AttackableUnits/AI/Minion.cs

```diff
@@ -57,7 +58,10 @@ public void PauseAi(bool b)
         public override void OnAdded()
         {
             base.OnAdded();
-            _game.PacketNotifier.NotifyMinionSpawned(this, Team);
+            if (!(Owner == null))
```

Comment:
Consider moving this check into `NotifyMinionSpawned`.

Reply:
Actually I see that you also check for `null` Owner in `NotifyMinionSpawned`. So is it that always `Minion` needs an owner? why? please explain about this cause I find this concept confusing.

Reply:
To quote the "Fix LaneMinions":
> Spawning LaneMinions with NotifyMinionSpawned causes them to stand still at their spawn position and teleport to targets (basically they don't act normal).
If there was no check for null owner in Minion.cs, this issue would occur. I think this is because NotifyMinionSpawned still has issues with updating minion position after spawn. If I make LaneMinions only spawn with NotifyLaneMinionSpawned, this issue does not occur, which is what the null check is for.

Reply:
Just disable in LaneMinion the call to `base.OnAdded();` (but check if that doesn't break anything - like things in IObjBase.

Reply:
Resolved here.